diff -Naur opl.orig/apps/demo_cn_console/linux.cmake opl.new/apps/demo_cn_console/linux.cmake
--- opl.orig/apps/demo_cn_console/linux.cmake	2016-11-02 07:57:06.000000000 -0400
+++ opl.new/apps/demo_cn_console/linux.cmake	2017-03-25 13:34:56.660213032 -0400
@@ -49,7 +49,7 @@
 IF (CFG_KERNEL_STACK_DIRECTLINK OR CFG_KERNEL_STACK_USERSPACE_DAEMON)
     SET (ARCH_LIBRARIES ${ARCH_LIBRARIES} pcap)
 ENDIF (CFG_KERNEL_STACK_DIRECTLINK OR CFG_KERNEL_STACK_USERSPACE_DAEMON)
-SET (ARCH_LIBRARIES ${ARCH_LIBRARIES} pthread rt)
+SET (ARCH_LIBRARIES ${ARCH_LIBRARIES} pthread rt wiringPi wiringPiDev)
 
 ################################################################################
 # Set architecture specific installation files
diff -Naur opl.orig/apps/demo_cn_console/src/app.c opl.new/apps/demo_cn_console/src/app.c
--- opl.orig/apps/demo_cn_console/src/app.c	2016-11-02 07:57:06.000000000 -0400
+++ opl.new/apps/demo_cn_console/src/app.c	2017-03-25 14:36:34.662652538 -0400
@@ -50,6 +50,7 @@
 #include <stddef.h>
 #include <stdio.h>
 
+#include <wiringPi.h>
 //============================================================================//
 //            G L O B A L   D E F I N I T I O N S                             //
 //============================================================================//
@@ -127,6 +128,11 @@
 
     ret = initProcessImage();
 
+    // Call the wiring Pi setup function to make GPIO Header Pin 17 as
+    // DigitalInput (Here Pin 0 is re-mapped as GPIO Header Pin 17)
+    wiringPiSetup ();
+    pinMode (0, OUTPUT);
+    
     return ret;
 }
 
@@ -168,12 +174,18 @@
     if (ret != kErrorOk)
         return ret;
 
+#if 0
     /* read input image - digital outputs */
     digitalOut_l = pProcessImageOut_l->digitalOut;
 
     /* setup output image - digital inputs */
     pProcessImageIn_l->digitalIn = digitalIn_l;
+#endif /* 0 */
 
+    // Once the slave receives the PDO data from the master, it writes
+    // the value to the GPIO Header Pin 17.
+    digitalWrite(0, pProcessImageOut_l->digitalOut);
+    
     ret = oplk_exchangeProcessImageIn();
 
     return ret;
